driver.go

```diff
@@ -113,9 +114,11 @@ func driver(cfg *config) trace.Driver {
 		},
 		OnConnNewStream: func(info trace.DriverConnNewStreamStartInfo) func(trace.DriverConnNewStreamDoneInfo) {
 			*info.Context = withTraceID(*info.Context)
-			if cfg.detailer.Details()&trace.DriverConnEvents == 0 {
+			if cfg.detailer.Details()&trace.DriverConnStreamEvents == 0 {
 				return nil
 			}
+			*info.Context = withReceiveMessagesCount(*info.Context, &atomic.Int64{})
```

Comment:
What do you think about single context creator without external counters?

Reply:
For example
```
*info.Context = withCounters(*info.Context)
```

Reply:
it's a good idea